wp-content/themes/bh/scss/_responsive.scss

```diff
@@ -2554,10 +2587,28 @@
 					height: 175px;
 				}
 			}
-			
+
+			h2.event-name {
+				font-size: 20px !important;
+			}
+
+			.event-meta {

```

Comment:
Please refer to my comment within event.scss regarding this new class named .event-meta.
In addition, structure here should be follow exactly the same structure of event.scss